x-pack/plugins/security_solution/public/common/containers/use_global_time/index.test.tsx

```diff
@@ -38,15 +38,21 @@ describe('useGlobalTime', () => {
   });
 
   test('clear all queries at unmount', () => {
-    const { rerender } = renderHook(() => useGlobalTime());
-    act(() => rerender());
+    const { unmount } = renderHook(() => useGlobalTime());
+    unmount();
     expect(mockDispatch.mock.calls[0][0].type).toEqual(
       'x-pack/security_solution/local/inputs/DELETE_ALL_QUERY'
     );
   });
 
   test('do NOT clear all queries at unmount', () => {
```

Comment:
NIT:
I think the test name could be more explicit like: "do NOT clear all queries at unmount when `clearAllQuery` is set to false."